input/postgres/relation_stats.go

```diff
@@ -36,8 +36,8 @@ SELECT c.oid,
 			 COALESCE(s.n_tup_hot_upd, 0),
 			 COALESCE(s.n_live_tup, 0),
 			 COALESCE(s.n_dead_tup, 0),
-			 s.n_mod_since_analyze,
-			 %s,
+			 COALESCE(s.n_mod_since_analyze, 0),
+			 COALESCE(%s, 0),
```

Comment:
Nitpick, but should we move that to `relationStatsSQLInsertsSinceVacuumFieldPg13`? (so for earlier versions, its just `0` instead of `COALESCE(0, 0)`.

Reply:
Yeah I was debating that, wondering what would be clearer. `COALESCE(0, 0)` returns 0, and this way we make it explicit that we should really COALESCE any columns that we may introduce next.
That's said, no strong opinion, happy to tweak to the other way if you think it reads better

Reply:
Fair, that's a good argument for having it here.